src/services/analyticsTracking.js

```diff
@@ -97,7 +97,7 @@ const setUserDimensions = (userId, role, groups) => {
   }
 
   if ((groups && groups.length) || (GAState.egoGroups && GAState.egoGroups.length)) {
-    let _groups = JSON.stringify({ egoGroups: groups || GAState.egoGroups });
+    let _groups = JSON.stringify({ egoGroups: groups[0] || GAState.egoGroups[0] });
```

Comment:
I'm getting this error here
![screen shot 2018-07-26 at 4 57 02 pm](https://user-images.githubusercontent.com/11821106/43288128-fdf518f2-90f4-11e8-803d-db4c834a55f4.png)